src/plugins/tagfilter/tagfilter.js

```diff
@@ -52,6 +54,11 @@ const componentName = "wb-tagfilter",
 				console.warn( componentName + ": You have no tagged items. Please add tags using the 'data-wb-tags' attribute." );
 			}
 
+			// Add accessibility to no result element
+			if ( noResultWrapper ) {
+				noResultWrapper.setAttribute( "aria-live", "polite" );
```

Comment:
Not sure if the only use of the `[aria-live=polite]` is enough to satisfy at [SC 4.1.3 Status messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html).

I do think the "Situation A" is applicable here because the status message advises on the results of an action.

According to the sufficient technique, it might require to use the role "status".

Also, to be tested with 1 AT, is the show/hide of the content block do get announced to AT user when:
1 - There is a few result
2 - There is no result (the block is displayed for the first time)
3 - There is a few result
4 - There is no result ( the block is re-displayed for a second time)

What about then the component do have 2 status content block? like the "Example 3". I do think the content block representing the status "no results" should be marked with the role "status" also.

Reply:
When I tested using the Mac screen-reader, I correctly got the message for no results every time (hiding, then showing, then hiding, etc.). However, I can add the role status to make it more obvious for the screen reader, and test.

Reply:
ok it is compatible with AT on mac. Have you compared against the SC 4.1.3 statement? Is it satisfied and applicable technique applied?

Reply:
Switched `aria-live="polite"` to `role="status"` which satisfies success criterion 4.1.3. However, while fixing this, I have discovered an issue with the current implementation which does not helpfully declare de changes in the DOM when a filter is selected. This issue is going to be fixed in an upcoming PR.